src/main/java/com/minecolonies/util/LanguageHandler.java

```diff
@@ -62,8 +63,10 @@ public static String getString(String key)
      * @param defaultValue the value to return if no key is found
      * @return Localized string
      */
+    @SuppressWarnings("deprecation")
```

Comment:
I would like to have compiler warnings be useful. That way when new deprecations or issues pop up they don't get lost in 50 warnings